q2_phylogenomics/plugin_setup.py

```diff
@@ -101,3 +105,79 @@
                 'based on quality scores using PRINSEQ-lite.',
     citations=[citations['schmieder_prinseq']]
 )
+
+filter_input = {'demultiplexed_sequences': 'The sequences to be trimmed.',
+                'database': 'Bowtie2 indexed database.'}
+filter_output = {'filtered_sequences': 'The resulting filtered sequences.'}
+
+filter_parameters = {
+    'n_threads': Int % Range(1, None),
+    'mode': Str % Choices([
+        'very-fast-local', 'fast-local', 'sensitive-local',
+        'very-sensitive-local', 'very-fast-global', 'fast-global',
+        'sensitive-global', 'very-sensitive-global']),
```

Comment:
What about another param for sensitivity:
```
'sensitivity': Str % Choices('very-fast', 'fast', 'sensitive', 'very-sensitive'),
'mode': Str % Choices('local', 'global'),
```

Reply:
👍